### PR TITLE
Components wireframe

### PR DESCRIPTION
**Pull request by Haywayaheadshot**
### What?

I created the wireframe for a book store app using react. I also added the components and pages for this app.

### Why?

This is to help users keep track of books they have read with detailed information on attributes of the book.

### How?
- I created the app using react
- I started with the mobile first approach
- I created a hamburger icon for the mobile menu
- I created the components of the app
- I created pages inside the components
- I routed every link to their respective pages

### CODE REVIEW

File: src/components/pages/Books.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      <section className="books-authors-inputs">
+        <input
+          placeholder="Book Title"
+          type="text"
+        />
+        <input
+          placeholder="Author"
+          type="text"
+        />
+        <button className="add-books" type="button">Add Book</button>
```
Comment (MahmoudBakr23) [inline src/components/pages/Books.js]: - [x] Please make sure to make a new specific component for the add book form

File: src/components/pages/Books.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './books.css';
+
+function Books() {
```
Comment (MahmoudBakr23) [inline src/components/pages/Books.js:6]: - [x] Please specify two different components, one for all the books list, and one component to render a single book

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @Haywayaheadshot 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done creating the components required so far.
- Nice job with functional components
- Descriptive PR
- Correct Linters

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, not criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

- Please fix the MIT link for a professional README file

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Haywayaheadshot** [comment]
@MahmoudBakr23 Thank you for reviewing my project. I will make the changes asap

File: src/components/pages/Books.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  return (
+    <div className="books-container">
+      <section className="received-api-books">
+        <ul className="new-book-entry-ul">
+          <li className="new-book-entry">New Book</li>
+          <li className="new-book-entry">New Book</li>
+          <li className="new-book-entry">New Book</li>
+          <li className="new-book-entry">New Book</li>
+        </ul>
```
Comment (AmaduKamara) [inline src/components/pages/Books.js]: Just as the previous reviewer highlighted earlier, you are to create a single Book.js component which you will be used in space of this `ul` tag here.
Note:

- [x] There should be a Books.js component to render a list of books(Which you now have 👍)
- [x] Next, you should create a single Book.js component that will be rendered in this Books.js component.
- The point here is that you will be having a list of books data later in an array that you'll be looping over and then render a Single Book.js per book to make a list of books. When you will have to loop later, book data is passed as a prop to the single Book.js component that you'll receive and render that data inside that Book.js.
- [x] It's a requirement that you have a single Book.js component that is rendered here in this Books.js component in space of the ul you are using. Kindly follow the instructions above to implement this 👍

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @Haywayaheadshot,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly 👍 
✅ Linter checks passed 👍 
✅ Used react-router-dom package for routing 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/pages/Books.js (hunk 3 above)
Comment (Haywayaheadshot) [reply src/components/pages/Books.js]: @AmaduKamara Thank you for reviewing my project.

I am a little confused as to what the requirement demands. Do you mean I should delete all other li's?

**Comment by AmaduKamara** [comment]
You should move the ul code into a Book.js components and then import that
component in the space where the ul is currently located

**Comment by Haywayaheadshot** [comment]
@AmaduKamara Thank you so much for your help.

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @Haywayaheadshot  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Haywayaheadshot** [comment]
@zainsadaqat Thank you for reviewing my project

**Pull request merged**